.scripts/postinstall.js

```diff
@@ -7,7 +7,9 @@
 const fs = require('fs');
 const path = require('path');
 
-if (process.env.NODE_ENV === 'dev') {
+// Only run this script if the module is being installed into
+// a "node_modules" folder.
+if (process.cwd().indexOf('node_modules') === -1) {
```

Comment:
We can remove this in favour of the es6 module approach + tree shaking. We can then have a single entry point in root that exports the components. (I'll add it in a PR soon)